### PR TITLE
feature: update README and add CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by bitpshr**
This pull request updates the `README` and adds a simple `CONTRIBUTING` guide. It's mostly a reorganization of existing information (moving contribution-specific content to a contributor-specific file to keep the `README` simple) with a few corrections after recent changes.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -45,17 +45,17 @@ threatening, offensive, or harmful.
 
 ## Scope
 
-This Code of Conduct applies both within project spaces and in public spaces
```
Comment (bitpshr) [inline CODE_OF_CONDUCT.md]: The changes in this file were the result of using the latest `covgen` command as per https://www.contributor-covenant.org/.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -45,17 +45,17 @@ threatening, offensive, or harmful.
...
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
 
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
+reported by contacting the project team at support@abstract.com. All
```
Comment (tommoor) [inline CODE_OF_CONDUCT.md]: 🙈

File: README.md [hunk 3]
```diff
@@ -1,44 +1,38 @@
...
+- [Documentation website](https://sdk.goabstract.com)
+- [Spectrum chat](https://spectrum.chat/abstract)
+- [Abstract help center](https://www.abstract.com/help/)
+- [Abstract support](https://www.abstract.com/help/contact/)
+- Still stuck? [File an issue!](https://github.com/goabstract/abstract-sdk/issues/new)
 
 ## Contributing
 
-This project is maintained by a team at [Abstract](https://www.goabstract.com). We are very willing to accept contributions, first please ensure there is a relavant [issue in the tracker](https://github.com/goabstract/abstract-sdk/issues) and an approach has been discussed before beginning to write code – this makes it more likely we will be able to accept your contribution and ensure nobody's time (especially yours!) is wasted.
+This project is maintained by a team at [Abstract](https://www.abstract.com). Outside contributions are highly encouraged, but please ensure that a relevant [issue](https://github.com/goabstract/abstract-sdk/issues) exists and that an approach has been discussed before begining to write code. This makes it more likely that your contribution will be accepted and ensures that your time is not wasted.
```
Comment (tommoor) [inline README.md]: Sp "begining"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Transpile the `src` directory into a `lib` directory using Babel:
+
+```sh
+yarn build
+```
+
+### Linting
+
+Lint an format all code using Prettier via ESLint:
```
Comment (tommoor) [inline CONTRIBUTING.md]: Sp "an" -> "and"

**Comment by bitpshr** [comment]
Thanks @tommoor, great catches. Should be ready for another review.

**Pull request merged**